docs/r2/get_started.ipynb

```diff
@@ -237,7 +301,7 @@
       },
       "cell_type": "markdown",
       "source": [
-        "<!-- <img class=\"tfo-display-only-on-site\" src=\"images/quickstart_model_fit.png\"/> -->"
+        "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/quickstart_model_fit.png?raw=1\"/>"
```

Comment:
This would make the image show up in the Colab notebook when it's running, right? So when the user runs the `%tensorboard` cell, they will see both the live TensorBoard and the image. How do we avoid that?

Reply:
Unfortunately, that's correct, at least at our current state. I filed b/124892469 and b/124898762 to request support for this in our import script to publish to tensorflow.org. @MarkDaoust 

Since `%tensorboard` is not supported in the TF docker image, we can't use it t render out the cells on import, so that's the reason for saving outputs here—and removing the output cell from %tensorboard while uncommenting the image.

The notebook is rendered in three places: tensorflow.org, the github previewer, and colab (static and when re-run). This temp solution works for each of these except when Colab is re-run, which, as you note, creates the double view.

Basically what we need is some conditional logic that says, if in this notebook environment, show this, otherwise, show this. But that's just not available across the display environments we're targeting.